nix/env.nix

```diff
@@ -37,6 +41,9 @@ mkShell {
       # These are needed for downloads and stuff
       cacert
       curl
+
+      # Provides libclang, which rocksDB needs
```

Comment:
Does it really? `LIBCLANG_PATH = "${libclang.lib}/lib";` is not enough?

Reply:
Good point, removed [here](https://github.com/tezos/riscv-pvm/compare/98853b5d4af2643d70304226fa757f74f9c24dad..b834b65bdaabbc5c6faa09c96eadafdcb6e288b1)